spec/features/draws/draw_student_report_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Draw student report' do
```

Comment:
Yea I just deleted all those changes, I think I can remake whatever I need for #51 when we do it for real.